packages/web-components/tests/e2e-storybook/cypress/integration/locale-modal/locale-modal.e2e.js

```diff
@@ -77,9 +77,10 @@ describe('dds-locale-modal | default', () => {
     const closeButton = cy
       .get('dds-locale-modal')
       .shadow()
-      .find('dds-expressive-modal-close-button');
-    closeButton
+      .get('dds-expressive-modal-close-button')
```

Comment:
I think we do actually want `.find` here. See https://docs.cypress.io/api/commands/get#Get-vs-Find.

Reply:
well. it wasn't working with find before, but i tried again and it does. <shrug> thanks!